README.md

```diff
@@ -7,13 +7,28 @@ directive (class property) using HTML5' LocalStorage.
 
 1. Download the library using npm or github: `npm install angular2-localstorage`
 2. Register the LocalStorage in your boot.ts:
-```typescript
-var appPromise = bootstrap(MyRootAppComponent);
+    ```typescript
+    var appPromise = bootstrap(MyRootAppComponent);
+    
+    // register LocalStorage, this registers our change-detection.
+    import {LocalStorageSubscriber} from 'angular2-localstorage/LocalStorageEmitter';
+    LocalStorageSubscriber(appPromise);
+    ```
+    or in your root component:
+    ```typescript
+    import {Component} from "angular2/core";
+    import {LocalStorageService} from "angular2-localstorage/LocalStorageEmitter";
+
+    @Componenet({
+        provider: [LocalStorageService]
+    })
+    export class AppRoot{
+        constructor(storageService: LocalStorageService){}
+    ...
+    }
+    ```
+
 
-// register LocalStorage, this registers our change-detection.
-import {LocalStorageSubscriber} from 'angular2-localstorage/LocalStorageEmitter';
-LocalStorageSubscriber(appPromise);
-```
 3. Use the `LocalStorage` decorator
 ```typescript
 import {LocalStorage} from "angular2-localstorage/LocalStorage";
```

Comment:
Once renamed we should use `import {LocalStorage, SessionStorage} from "angular2-localstorage/WebStorage";`